docs/source/release_notes/release-notes.rst

```diff
@@ -53,6 +53,15 @@ develop
            This prevents the operation from interfering with concurrently running Postgres backups.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1616>`__)
 
+    *    - |new|
+         - Users can now create a Docker image and run containers of the Timelock Server, by running ``./gradlew timelock-server:dockerTag``.
+           This can be useful for quickly spinning up a Timelock instance (e.g. for testing purposes).
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1661>`__)
+
+    *    - |fixed|
+         - AtlasDB :ref:`CLIs <clis>` can now work with a Timelock block, and will contact the relevant Timelock server for timestamps or locks in this case.
```

Comment:
Yep. The throwing behaviour only occurs when you run the CLIs as part of the Dropwizard bundle, actually - have clarified that!